### PR TITLE
fix: deployments created in preview version don't work in GA version

### PR DESCRIPTION
**Pull request by philasmar**
*Issue #, if available:*
DOTNET-6124

*Description of changes:*
The issue was caused by a breaking change we introduced when we changed the Type of the Security group settings in the fargate recipe. The type changed from `string` to `List` which broke the workflow when the tool tried to apply previous settings on redeployments. This change skips broken settings when applying previous settings and it gives users a message that we were unable to set that setting. I also added a validation error when this happens so that the VS Toolkit tells the user that there is an issue with one of the settings, giving the user the option to either fix the issue before deployment.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/AWS.Deploy.Orchestration/Orchestrator.cs
Comment (normj) [inline src/AWS.Deploy.Orchestration/Orchestrator.cs:170]: Change `LogInfoMessage` to `LogErrorMessage`.

**Review by normj — APPROVED** [review]
Approving assuming you make the one minor change suggested. Preemptively approving since I'm going on PTO soon and I don't want you blocked by me.

**Review by ashovlin — APPROVED** [review]

File: src/AWS.Deploy.Orchestration/Orchestrator.cs
Comment (philasmar) [reply src/AWS.Deploy.Orchestration/Orchestrator.cs]: Fixed

**Pull request merged**
